src/InfrastructureSystems.jl

```diff
@@ -63,7 +63,8 @@ end
 set_time_series_container!(value::InfrastructureSystemsComponent) = nothing
 
 get_name(value::InfrastructureSystemsComponent) = value.name
-set_name!(value::InfrastructureSystemsComponent, name) = value.name = name
+set_name_internal!(value::InfrastructureSystemsComponent, name) = value.name = name
```

Comment:
```suggestion
function set_name_internal!(value::InfrastructureSystemsComponent, name) 
    value.name = name
    return
    end
```

otherwise the name is returned.

Reply:
Good point. Thanks.